cvmfs/quota_posix.h

```diff
@@ -372,6 +385,29 @@ class PosixQuotaManager : public QuotaManager {
    * Used in the destructor to steer closing of the database and so on.
    */
   bool initialized_;
+
+  /**
+   * Used in DoCleanup to exclude currently used files from eviction
+   */
+  // TODO(gchr): it would be faster if it was a std::set. Needs a comparison
+  // operator for shash::Any
+  pthread_mutex_t *lock_open_files_;
+  std::vector<shash::Any> open_files_;
+
+  bool cleanup_unused_first_;
+  std::vector<std::string> mountpoints_;
+
+  std::vector<shash::Any> CollectGroupsHashes();
```

Comment:
I still don't like this name. What about 
```suggestion
  std::vector<shash::Any> CollectAllOpenHashes();
```
?